src/lib/nvme_pt_ioctl.h

```diff
@@ -375,7 +375,7 @@ int opal_setuplr_pt(struct sed_device *dev, const char *key, uint8_t key_len,
 int opal_lock_unlock_pt(struct sed_device *dev, const struct sed_key *key,
 		enum SED_ACCESS_TYPE lock_type);
 
-int opal_set_pwd_pt(struct sed_device *dev, const struct sed_key *old_key,
+int opal_set_pwd_pt(struct sed_device *dev, enum SED_AUTHORITY auth, const struct sed_key *old_key,
```

Comment:
Need to update the protocol in nvme_pt_ioctl.c as well

Reply:
Good catch. posted missing file in the second commit